llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1395,6 +1395,19 @@ def NoConditionalMoveFusion  : Predicate<"!Subtarget->hasConditionalMoveFusion()
 def TuneMIPSP8700
     : SubtargetFeature<"mips-p8700", "RISCVProcFamily", "MIPSP8700",
                        "MIPS p8700 processor">;
+def FeatureVendorMIPSCMove : SubtargetFeature<"xmipscmove", "HasVendorMIPSCMove",
```

Comment:
FeatureVendor* should be in the vendor extension section of this file not the tuning section.